setup.cfg

```diff
@@ -51,9 +51,8 @@ install_requires =
     confection>=0.0.1,<1.0.0
     # Third-party dependencies
     setuptools
-    numpy>=1.19.0,<3.0.0; python_version < "3.9"
-    numpy>=1.19.0,<3.0.0; python_version >= "3.9"
-    pydantic>=1.7.4,!=1.8,!=1.8.1,<3.0.0
+    numpy>=2.0.0,<3.0.0
```

Comment:
@honnibal Why was numpy>=2.0.0 updated here? This code change ultimately was included in https://github.com/explosion/thinc/releases/tag/release-v8.3.6 however in https://github.com/explosion/thinc/releases/tag/release-v8.3.3 the point was made that the package runtime could be indeed compatible with Numpy 1 and 2 (while build process requires Numpy 2).

> Widen numpy pin, allowing versions across v1 and v2. Previously I had thought that if I build against numpy v2, I couldn't also have v1 as a runtime dependency. This is actually incorrect, so we can widen the numpy pin